tap_stella/sync.py

```diff
@@ -44,30 +45,40 @@ def sync_qa(client, stream, state):
 
         # write one or more rows to the stream:
         singer.write_records(stream.tap_stream_id, rows)
-        singer.write_state({stream.tap_stream_id: new_bookmark})
+    if new_bookmark:
```

Comment:
Can we just use `singer.write_bookmark` instead?

https://github.com/singer-io/singer-python/blob/master/singer/bookmarks.py

Reply:
writing bookmarks require a replication key and the shape of the state after writing will look like:
```
{
    "bookmarks": {
        stream_id: {
            replication_key: bookmark_value
        }  
    }
}
```
this is also fine, but we'd need to alter the tap to do it this way. Ill include a TODO to update this in the future.
for now I think this is fine